apps/api/src/variable/service/variable.service.ts

```diff
@@ -413,49 +413,79 @@ export class VariableService {
     sort: string,
     order: string,
     search: string
-  ) {
-    // Check if the user has the required authorities in the project
-    await this.authorityCheckerService.checkAuthorityOverProject({
-      userId: user.id,
-      entity: { id: projectId },
-      authority: Authority.READ_VARIABLE,
-      prisma: this.prisma
-    })
+  ): Promise<
+    { environmentName: string; environmentId: string; variables: any[] }[]
```

Comment:
You can refactor this return type into `variable.types.ts`.

Do a similar thing for secrets.